Whats New Chrome 105 Spanish Translation (#3507)
GoogleChrome/developer.chrome.com: site/_includes/partials/devtools/es/whats-new.md

### Chrome 105 {: #chrome105 }

> * [Reproducción paso a paso en la grabadora](/es/blog/new-in-devtools-105/#recorder)

Reviewer: ```suggestion
* [Reproducción paso a paso en el panel Recorder](/es/blog/new-in-devtools-105/#recorder)
```

> * [Soporte de eventos de ratón en el panel de la grabadora](/es/blog/new-in-devtools-105/#recorder-hover)

Reviewer: ```suggestion
* [Soporte de eventos de ratón en el panel Recorder](/es/blog/new-in-devtools-105/#recorder-hover)
```

> * [Largest Contentful Paint (LCP) en el panel de percepción de Rendimiento](/es/blog/new-in-devtools-105/#lcp)

Reviewer: I'd suggest that we refer to strings that aren't translated in DevTools as they are displayed.

```suggestion
* [Largest Contentful Paint (LCP) en el panel Performance Insights](/es/blog/new-in-devtools-105/#lcp)

```
